plugins/basic/nexus-repository-nuget/src/main/java/com/sonatype/nexus/repository/nuget/internal/proxy/NugetProxyGalleryFacet.java

```diff
@@ -128,12 +128,17 @@ protected void doDestroy() throws Exception {
 
   @Override
   public int count(final String operation, final Map<String, String> parameters) {
+    Iterable<Repository> proxyRepositories = getProxyRepositories();
     final List<Integer> remoteCounts = passQueryToRemoteRepos(nugetQuery(operation, parameters),
-        getProxyRepositories(),
+        proxyRepositories,
         new CountFetcher(fetcher));
-
-    final int hostedCount = count(operation, parameters, getHostedRepositories());
-
+    log.debug("Counts from proxyRepositories: {} are: {}", proxyRepositories, remoteCounts);
+    
+    Iterable<Repository> hostedRepositories = getHostedRepositories();
+    final int hostedCount = hostedRepositories.iterator().hasNext()
```

Comment:
you could also use `!Iterables.isEmpty(...)` which might be slightly more readable